content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -118,81 +118,140 @@ On your validator:
+
+
+### Connect using public hubs
+
+This configuration connects your validator to the network using two public hubs. This configuration is similar to [connecting using proxies you run yourself](#connect-using-proxies), but instead you connect through public hubs. Successful public hubs embody the following traits:
+
+- Good bandwidth.
+
+- Connections with a lot of reliable peers.
+
+- Ability to relay messages reliably.
+
+**Pros:**
+
+  - Enables you to trust your validator's peers because you only connect to well-known servers with a high reputation.
+
+  - Provides easy access to safe connections to the network.
+
+  - Compared to [connecting using proxies](#connect-using-proxies), may be less likely to cause your validator to disconnect from the network due to a simultaneous peer outage.
+
+**Cons:**
+
+  - Uses the default (most popular) public hubs, so it's distinctly possible that a public hub may be too busy to provide your validator with a connection to the network. Assuming that you can solely rely on a default public hub to provide your validator with a connection to the network may not be the best idea.
+
+    To help avoid this issue, use more public hubs; the more the better. It can also help to use non-default hubs, which are less likely to be busy helping new servers find other peers.
+
+_**To connect your validator to the network using public hubs, set the following configurations in your validator’s `rippled.cfg` file:**_
+
+1. Include the following `[ips_fixed]` stanza. The two values, `r.ripple.com 51235` and `zaphod.alloy.ee 51235`, are default public hubs. This stanza tells `rippled` to always attempt to maintain peer connections with these public hubs.
+
+        [ips_fixed]
+        r.ripple.com 51235
+        zaphod.alloy.ee 51235
+
+    **Caution:** This configuration connects your validator to the network using default public hubs. Because these are the _default_ public hubs, they may sometimes be too busy to provide your validator with a connection to the network. To help avoid this issue, connect to more public hubs and, even better, connect to non-default public hubs.
+
+    You can include the IP addresses of other `rippled` servers here, but _**only**_ if you can expect them to:
+
+      - Relay messages without censoring.
+      - Stay online consistently.
+      - Not DDoS you.
+      - Not try to crash your server.
+      - Not publish your IP address to strangers.
+
+2. Include the following `[peer_private]` stanza and set it to `1`. Enabling this setting instructs your validator’s peers not to broadcast your validator’s IP address. This setting also instructs your validator to connect to only the peers configured in your `[ips_fixed]` stanza. This ensures that your validator connects to and shares its IP with only peer `rippled` servers you know and trust.
+
+    **Warning:** Be sure that you don't publish your validator's IP address in other ways.
```

Comment:
I suggest moving the warning down so it doesn't split the instructions apart from the code sample